client/src/js/components/bhAccountSelect.js

```diff
@@ -36,6 +37,9 @@ function AccountSelectController(Accounts, AppCache, $timeout, bhConstants, $sco
     // cache the title account ID for convenience
     $ctrl.TITLE_ACCOUNT_ID = bhConstants.accounts.TITLE;
 
+    // To filter accounts by class of accounts
+    $ctrl.classe = $ctrl.classe || '';
```

Comment:
I really think this is a duplication of `account_type_id`.  Let's hold off on this dependency until we are certain we want to use `classe` instead of `account_type`.  After all, users can view and modify the `type_id` on the account page, but they cannot modify the `classe`.